src/test/scala/examples/LoadMemoryFromFileSpec.scala

```diff
@@ -0,0 +1,77 @@
+// See LICENSE for license details.
+
+package examples
+
+// See LICENSE for license details.
+
+import java.nio.file.{Files, Paths}
+import java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+import chisel3._
+import chisel3.iotesters.PeekPokeTester
+import chisel3.util.loadMemoryFromFile
+import firrtl.FileUtils
+import org.scalatest.{FreeSpec, Matchers}
+
+//noinspection TypeAnnotation
```

Comment:
I don't think this is an editor specific setting, it is a standard directive to the scala style-checker which we are supposed to be supporting.

Reply:
See what you think. I had never seen this before. From what I could dig up online, I concluded that it was IntelliJ adding code comments:
  - https://stackoverflow.com/questions/7798908/disable-warning-in-intellij-for-one-line
  - https://intellij-support.jetbrains.com/hc/en-us/community/posts/206280839--SuppressWarnings-versus-noinspection

Scalastyle directives (which I thought, perhaps incorrectly, all started with `scalastyle`) I'm fine with, on the other hand.

Reply:
You are right, I didn't think it through I will change it to the scalastyle directive, sorry.

Reply:
I'll just get rid of it for now. It's an unfortunate scalastyle collision that I don't think there is a great way to fix. I'm not sure that we should change the style rules for this situation because other than IO I think this is a good thing to flag.